web_widget_timepicker/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================
+Timepicker widget in form view
+==============================
+
+This module defines a timepicker widget, to be used with either char fields
+or (function) fields of type character. Use ``widget='timepicker'`` in your form view
```

Comment:
I find the char field choice strange: usually a Float field is used to enter time, and a widget `float_time` exists.
Why change this standard?